src/routes/productRoutes.ts

```diff
@@ -36,5 +38,5 @@ router.delete("/api/products/:id", authenticateJWT, isAdmin, validateDeleteProdu
 router.get("/api/products/:id/ratings", validateGetProductRatings, getProductRatings);
 router.get("/api/products/new-arrivals", validateGetNewArrivals, getNewArrivals);
 router.get("/api/products/by-brand/:brandId", validateGetBrandProduct, validateGetAllProducts, getProductsByBrandController);
-
+router.get("/api/products/:id/price-after-discount", authenticateJWT,validateGetPriceAfterDiscount, getProductPriceAfterDiscount);
```

Comment:
Are you sure about the structure of this rout? 

@Lojain-Abdalrazaq  what you think?

Reply:
I think its clear, and similar to the routes of the product table, for example: "/api/products/:id/ratings". 

Do you have any suggestions or ideas for improving it? @AlaaYa7ia